public/pages/utils/anomalyResultUtils.ts

```diff
@@ -356,7 +406,8 @@ export const getBucketizedAnomalyResultsQuery = (
   startTime: number,
   endTime: number,
   interval: number,
-  detectorId: string
+  detectorId: string,
+  entity: Entity | undefined = undefined
 ) => {
   const fixedInterval = Math.ceil(
     (endTime - startTime) / (interval * MIN_IN_MILLI_SECS * MAX_DATA_POINTS)
```

Comment:
Could you help me understand the formula?  I thought we want to find out if the interval is smaller than MAX_DATA_POINTS or not? So

```
let fixedInterval = Math.ceil((endTime - startTime) / (interval * MIN_IN_MILLI_SECS));
if (fixedInterval > MAX_DATA_POINTS) {
    fixedInterval = MAX_DATA_POINTS;
}
```

Reply:
I think above [comment](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/350#discussion_r550337962) can explain it.